source/ed/connectors/fusio_parser.h

```diff
@@ -252,10 +252,10 @@ struct TripPropertiesFusioHandler : public GenericHandler {
 namespace grid_calendar {
 struct PeriodFusioHandler : public GenericHandler {
     PeriodFusioHandler(GtfsData& gdata, CsvReader& reader) : GenericHandler(gdata, reader) {}
-    int calendar_c, begin_c, end_c;
+    int id_c, begin_c, end_c;
     void init(Data&);
     void handle_line(Data& data, const csv_row& line, bool is_first_line);
-    const std::vector<std::string> required_headers() const { return {"calendar_id", "begin_date", "end_date"}; }
+    const std::vector<std::string> required_headers() const { return {"end_date"}; }
```

Comment:
```suggestion
    const std::vector<std::string> required_headers() const { return {"grid_calendar_id", "start_date", "end_date"}; }
```

Reply:
Why removing the header in all the objects ? They should be required according to the specs.

Reply:
Because of the backward compatibility., we have to support new and old names. 
We can't freeze the header.